lib/rails_admin/config/actions/ips_mismatch.rb

```diff
@@ -15,8 +15,11 @@ class IpsMismatch < Base
             @collection = @object.solutions
               .where('solutions.ips != users.registration_ips')
               .group('users.id')
-              .pluck('users.registration_ips', 'users.secret', 'array_agg(solutions.id)')
-              .map { |ips, secret, solution_ids| [ips, secret, Solution.includes(:task).where(id: solution_ids)] }
+              .pluck('users.registration_ips', 'users.judge_secret', 'array_agg(solutions.id)')
+              .map do |ips, judge_secret, solution_ids|
+              [ips, judge_secret,
+                Solution.includes(:task).where(id: solution_ids)]
```

Comment:
If you are going to split it to multiple lines, let's then use full name `registration_ips` for the first argument.

Also, is the indentation missing here?